### ISSUE TITLE
Cleaning of Maps

### ISSUE THREAD

**Issue by xkae591** [itsnicksia/wizardry-daphne-guide#14]
Thanks for making the web guide! Really appreciate the help I've gotten thus far and wish to give some back.

I've noticed that the maps provided in the guide are sometimes difficult to use, and some of them lacks quest elements such as mob locations and quest markers. These would be very helpful not only to reduce clunk, but also to make it more user-friendly when using the maps for navigation.

There are some tools around for Wizardry, such as [Gridmonger](https://gridmonger.johnnovak.net/), that provides a clean look. These applications save maps in an editable and sharable format so that anyone can use the them as a base for further edits.

Here's a sample I've done for Beginning Abyss to illustrate my point:

![image](https://github.com/user-attachments/assets/34673159-1ea2-4da1-bb37-1f13089c4cf9)

As I've already done some on my end to track my navigation during my play-through, please do let me know if you would be interested in using my files.

Thanks again for making the web guide! Looking forward to greater things for this relatively obscure (but fun) mobile game.

**Comment by TheAxolotl-gh** [comment]
@maybeLynd @itsnicksia @Koston-0xDEADBEEF Definitely curious what you three think of this.

On one hand, I like the idea of using in-game images, but on the other hand, this looks pretty slick and tying it to known Wizardry tools is a fun throwback.

**Comment by Koston-0xDEADBEEF** [comment]
Can you provide the map files? I would like to take a look at the Gridmonger tool and see how much effort it requires to maintain maps in this format and if exporting map updates can be automated.

**Comment by xkae591** [comment]
Certainly! Attached is the Gridmonger file, as requested. It's partially maintained and worked on-the-fly, so do forgive me for the incomplete maps and inconsistent icons.

The [user manual](https://gridmonger.johnnovak.net/manual/contents.html) explains most of the features.

There are other well-accredited tools out there such as [DungeonDraft](https://dungeondraft.net/), [DungeonScrawl](https://app.dungeonscrawl.com/), and [RPTools](https://www.rptools.net/) if you prefer cross-application compatibility using .VTT files and power-features like map designing tools.

Personally, once you get used to the keyboard shortcuts, map creation becomes very fast.

[Map.zip](https://github.com/user-attachments/files/18032254/Map.zip)

**Comment by Koston-0xDEADBEEF** [comment]
I took a look at the Gridmonger tool and your maps. I like the tool a lot, it's very cleanly designed and implemented. It's currently lacking a feature for exporting maps as images, but according to the maintainer this is also planned. It already has nice theme support, enabling the maps to fit seamlessly into the guide.

The relevant question is, are you willing to maintain the maps themselves? I can handle creating a suitable theme and including them in the guide, but also drawing everything by myself is too time consuming.

**Comment by xkae591** [comment]
I don't foresee any difficulties on my part in maintaining the maps at the moment, and I'd certainly be willing to undertake the responsibility.

Some maps have recently been posted in the #guide-contributions channel (specifically Trade Waterways 6 and Arena 2 as the guide lacked adequate information about those at the time of writing). I'll make the remaining maps in due time, but the more straightforward maps will have less priority.

Feel free to let me know if yall have specific requests, such as making a clean map separate from mob-identification markers, adding quest markers, etc.! I'll try to fulfill those asap before moving on to other projects.

Do bring this conversation to Discord if that's more convenient. Thanks.

**Comment by TheAxolotl-gh** [comment]
Here are a few things to think about:

1. I think these maps are great, particularly when it comes to something like the Trade Waterways where you can show the flooding areas and other potential future hazards, however I think it's going to be a bit tedious to try and maintain multiple versions of a floor's map, and you know we'll get questions and comments about someone having a different version of the map than we have out there. It's currently a really easy fix to just upload their image rather than re-draw the entire new version. Do we flat out say we're not going to maintain individual versions anymore, and your mileage may vary with the maps themselves?

2. What happens if you quit the game or stop maintaining the maps?

3. Do we include both the Gridmonger maps AND image maps? How do we standardize them? Granted, our maps right now aren't standardized, but since they're all game images, there's some element of standardization in there.

**Comment by xkae591** [comment]
I agree with the points Koston has made. To add on,

**On standardization and presentation.**
Gridmonger does have a *region tool* that could be particularly useful in demarcating possible altercations to maps (see screenshot attached). Thus, simply editing those regions is enough, and users don't necessarily have to remap entire maps. This could help make the content tabs (suggested in Koston's 2nd point) easier to navigate.

![image](https://github.com/user-attachments/assets/a14de0c8-37a6-4392-8a21-99ead86350f9)

**On maintaining maps.**
There's no getting around the fact that people come and go either. While there are definitely mapping tools out there with collaboration in mind (if we're really pushing it, Google Sheets could even sound nice lol), I believe just sharing Gridmonger map files (the `.gmm` files) is enough.

The benefits gained from using our own maps will depend on the purpose of the website.
- If the website is meant to provide only the relevant information that points lost users to the right direction, then perhaps not having our own maps would be economically better.
- On the other hand, if the website wants to document as much of WVD as possible, in the best way possible, then having the flexibility to edit maps based on specific objectives would be much better than taking in-game screenshots and highlighting points in Photoshop. For example, you can achieve something like [this](http://wizardryarchives.com/maps/WizardryIMaps.html) better with custom maps than without.

I totally understand the apprehensiveness of not wanting to add unnecessary dependencies to the already mundane task of maintaining a web guide. Won't be pushing this project further if that's the case, since I can neither guarantee my faithful involvement nor potential successors in the event I stop maintaining the maps.
